### PR TITLE
add copyrights from IOTPUSH project

### PR DESCRIPTION
**Pull request by kaspar030**
### Contribution description

Over the last years, most of my RIOT work was done as part of the IOTPUSH project, a joint project between Inria and Freie Universität Berlin.

This PR updates some copyright statements accordingly, adding both Freie Universität Berlin and Inria.

### CODE REVIEW

File: sys/include/event/timeout.h [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /*
- * Copyright (C) 2017 Kaspar Schleiser <kaspar@schleiser.de>
+ * Copyright (C) 2017 Inria
+ *               2017 Freie Universität Berlin
+                 2017 Kaspar Schleiser <kaspar@schleiser.de>
```
Comment (emmanuelsearch) [inline sys/include/event/timeout.h]: nit: star missing?

File: sys/include/event/callback.h [hunk 2]
```diff
@@ -1,5 +1,7 @@
 /*
- * Copyright (C) 2017 Kaspar Schleiser <kaspar@schleiser.de>
+ * Copyright (C) 2017 Inria
+ *               2017 Freie Universität Berlin
+                 2017 Kaspar Schleiser <kaspar@schleiser.de>
```
Comment (emmanuelsearch) [inline sys/include/event/callback.h]: nit: star missing?

File: sys/include/event.h [hunk 3]
```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2017 Kaspar Schleiser <kaspar@schleiser.de>
+ * Copyright (C) 2017 Inria
+                 2017 Kaspar Schleiser <kaspar@schleiser.de>
```
Comment (emmanuelsearch) [inline sys/include/event.h]: nit: star missing?

File: sys/event/timeout.c [hunk 4]
```diff
@@ -1,5 +1,7 @@
 /*
- * Copyright (C) 2017 Kaspar Schleiser <kaspar@schleiser.de>
+ * Copyright (C) 2017 Inria
+ *               2017 Freie Universität Berlin
+                 2017 Kaspar Schleiser <kaspar@schleiser.de>
```
Comment (emmanuelsearch) [inline sys/event/timeout.c]: nit: star missing?

File: sys/event/callback.c [hunk 5]
```diff
@@ -1,5 +1,7 @@
 /*
- * Copyright (C) 2017 Kaspar Schleiser <kaspar@schleiser.de>
+ * Copyright (C) 2017 Inria
+ *               2017 Freie Universität Berlin
+                 2017 Kaspar Schleiser <kaspar@schleiser.de>
```
Comment (emmanuelsearch) [inline sys/event/callback.c]: nit: star missing?

File: core/clist.c [hunk 6]
```diff
@@ -1,5 +1,7 @@
 /*
- * Copyright (C) 2017 Kaspar Schleiser <kaspar@schleiser.de>
+ * Copyright (C) 2017 Inria
+ *               2017 Freie Universität Berlin
+                 2017 Kaspar Schleiser <kaspar@schleiser.de>
```
Comment (emmanuelsearch) [inline core/clist.c]: nit: star missing?

**Comment by kaspar030** [comment]
Thanks @emmanuelsearch, I've added the missing stars and force-pushed the amended changes. Please take another look!

**Comment by emmanuelsearch** [comment]
No more comments!

**Comment by emmanuelsearch** [comment]
and go!

**Pull request merged**